core/common/src/main/java/alluxio/grpc/GrpcChannelKey.java

```diff
@@ -56,17 +56,25 @@ private GrpcChannelKey(AlluxioConfiguration conf) {
   }
 
   /**
-   * Creates a {@link GrpcChannelKey}.
+   * Creates a unique identifier for a gRPC channel.
+   * <p>
+   * Is used to identify a {@link alluxio.grpc.GrpcChannel}
```

Comment:
I would improve this sentence: "Identifies the {@link alluxio.grpc.GrpcChannel} using a {@link GrpcChannelKey} in an unique and unambiguous way.".
Avoid starting methods with "Is, Are, Was, Were"

Reply:
Fixed.